plugins/module_utils/attachment.py

```diff
@@ -27,8 +27,7 @@ def _path(*subpaths):
 
 class AttachmentClient:
     def __init__(self, client, batch_size=10000):
-        # 10000 records is default batch size for ServiceNow Attachment REST API, so we also use it
-        # as a default.
+        # 10000 records is default batch size for ServiceNow Attachment REST API, so we also use it as a default.
```

Comment:
I would suggest keeping this as it is.

Reply:
I agree, but I think I already reverted this in some previous commits, so the code is already as suggested